spec/models/group_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.describe Group, type: :model do
+  subject do
+    @user = User.create!(name: 'someone', email: 'user@user.com', password: 'password', id: 1)
+    @icon_file = fixture_file_upload(Rails.root.join('spec', 'fixtures', 'files', 'test.png'), 'image/png')
+    @group = Group.create(name: 'stationary', icon: @icon_file, user_id: @user.id)
+  end
+
+  before { subject.save }
+
+  it 'is valid with valid attributes' do
+    expect(subject).to be_valid
+  end
+
+  it 'is not valid if any empty attribute' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'name should be present' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'name should not be too short' do
+    subject.name = 'a'
+    expect(subject).to_not be_valid
+  end
+
+  it 'name should not be too long' do
+    subject.name = 'a' * 301
+    expect(subject).to_not be_valid
+  end
+
+  it 'name should not be empty' do
+    subject.name = ' '
+    expect(subject).to_not be_valid
+  end
+
+  it 'name should have valid value' do
+    expect(subject.name).to eql 'stationary'
+  end
+
+  it 'user should be present' do
+    subject.user = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'user_id should be present' do
+    subject.user_id = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'user_id not be empty' do
+    subject.user_id = ' '
+    expect(subject).to_not be_valid
+  end
```

Comment:
- [ ] Great job working on the tests so far but As I can see after running the tests this group tests it doesn't pass please could you solve the problem so you let all your tests just pass 🔥💯🤩✅✅✅


![image](https://user-images.githubusercontent.com/58553711/221380705-91e9f787-a64a-4fe3-86b4-7c342844aa61.png)